tests/cache/tests.py

```diff
@@ -1366,6 +1366,19 @@ def test_get_does_not_ignore_non_filenotfound_exceptions(self):
             with self.assertRaises(IOError):
                 cache.get('foo')
 
+    def test_empty_cache_file_does_not_throw_exception(self):
+        cache.set('foo', 'bar')
+        self.assertEqual(cache.get('foo'), 'bar')
+        with open(cache._key_to_file('foo'), 'wb') as fh:
+            fh.write(b'')
+        with open(cache._key_to_file('foo'), 'rb') as fh:
+            self.assertTrue(cache._is_expired(fh))
```

Comment:
Recently, we've been using `self.assertIs(..., True)` as `assertTrue()` could hide a bug.